src/modm/platform/can/stm32/can.hpp.in

```diff
@@ -108,7 +108,7 @@ public:
 	template< class SystemClock, bitrate_t bitrate=kbps(125), percent_t tolerance=pct(1) >
 	static inline bool
 	initialize(	uint32_t interruptPriority, Mode startupMode = Mode::Normal,
-				bool overwriteOnOverrun = true)
+				bool overwriteOnOverrun = true) [[nodiscard]]
```

Comment:
Is it not to be used like this?
```c++
	template< class SystemClock, bitrate_t bitrate=kbps(125), percent_t tolerance=pct(1) >
 	static inline [[nodiscard]] bool
 	initialize(	uint32_t interruptPriority, Mode startupMode = Mode::Normal
				bool overwriteOnOverrun = true)
```

Reply:
Position almost doesn't matter. Tried several.
Also wasn't able to find an explanation if this attribute is valid on class members or not. Don't see a reason why not.

Reply:
I was a little surprised how badly it's documented. But if it works this way, fine by me.